drivers/can/Kconfig.stm32

```diff
@@ -16,7 +16,16 @@ config CAN_MAX_FILTER
 	int "Maximum number of concurrent active filters"
 	depends on CAN_STM32
 	default 5
-	range 1 56
+	range 1 112
 	help
-	  Defines the array size of the callback/msgq pointers.
+	  Defines the array size of the callback pointers.
 	  Must be at least the size of concurrent reads.
+
+config CAN_FILTER_SPLIT
```

Comment:
Needs to go to the device-tree, because there are SoC which has two dual-port CAN and one single port.